frontend/src/app/components/channel/small/small.controller.ts

```diff
@@ -81,13 +81,13 @@ export class SmallChannelController extends BaseChannelController {
   }
 
   // TODO: Handle HTTP errors and rollback to old state in case of an error
-  private putCompositeToSlot(index: number, program: boolean): ng.IPromise<any> {
+  private putCompositeToSlot(index: number, program: boolean, force: boolean = false): ng.IPromise<any> {
     var deffered = this.$q.defer();
 
     var slotName = this.getSlotName(program);
 
     // Composite already in slot ?
-    if (index === this.compositeIndex[slotName]) {
+    if (index === this.compositeIndex[slotName] && !force) {
```

Comment:
Maybe to keep current composites instead of composite indexes, then no need to `force` parameter
anyway would be good if we could take these issues together